docs/issuance-guide.mdx

```diff
@@ -167,6 +167,10 @@ If the user does not complete the partner onboarding process within 5 minutes, t
 
 If this is an experience you would like to provide for your partners immediately after credential issuance, please let us know so we can provide you with your partner's static UUID and enable this capability for your <Tip type="brand">brand</Tip>.
 
+:::tip
+You can check the partner's pre-defined <Tip type="request">credential requests</Tip> by calling [`/credentialsRequests/{partnerUuid}`](/api-overview/#get-partner-credential-requests).
+:::
+
```

Comment:
```suggestion
```

Reply:
I would move this between line 147 and 148. (Also, could you please add a new line above the current line 148? Spacing looks funny on the site.

Reply:
Between 147 and 148 feels strange

<img width="981" alt="image" src="https://github.com/UnumID/UnumID.github.io/assets/11665444/f0dadb83-a413-4277-aa39-bc05c818f73a">

Maybe below both looks better

<img width="981" alt="image" src="https://github.com/UnumID/UnumID.github.io/assets/11665444/cf2e2424-7e9e-4b54-accd-638b2686d4db">

but I personally prefer bellow the example hahah

<img width="981" alt="image" src="https://github.com/UnumID/UnumID.github.io/assets/11665444/096a152a-039f-4599-a0b7-e45c5796070f">


Maybe this shouldn't be a tip box, maybe we can add this content to text ?

@liamhalemccarty any thoughts?

Reply:
I actually agree with Ray that it's best between 147 and 148, since it's near the mention of "credential requests". It could make sense as part of the text too, but it's a side comment and shouldn't distract from the main flow of the text.

Reply:
So I'll definitely put the tip box between those lines